extensions/wikia/AchievementsII/Ach_setup.php

```diff
@@ -331,17 +329,16 @@ function AchTest() {
  *
  * @author Macbre
  */
-function Ach_isBadgeImage($destName, $checkUserRights = false) {
+function Ach_isBadgeImage( $destName, $checkUserRights = false ) {
 	global $wgUser;
 
-	$isBadge = strlen($destName) > strlen( ACHIEVEMENTS_BADGE_PREFIX ) && stripos($destName, ACHIEVEMENTS_BADGE_PREFIX) === 0;
-	$isSponsoredBadge = strlen($destName) > strlen( ACHIEVEMENTS_HOVER_PREFIX ) && stripos($destName, ACHIEVEMENTS_HOVER_PREFIX) === 0;
+	$isBadge = strlen( $destName ) > strlen( ACHIEVEMENTS_BADGE_PREFIX ) && stripos( $destName, ACHIEVEMENTS_BADGE_PREFIX ) === 0;
+	$isSponsoredBadge = strlen( $destName ) > strlen( ACHIEVEMENTS_HOVER_PREFIX ) && stripos( $destName, ACHIEVEMENTS_HOVER_PREFIX ) === 0;
 
-	if ($checkUserRights) {
+	if( $checkUserRights ) {
 		$matches = ( $isBadge && !( $wgUser->isAllowed( 'platinum' ) || $wgUser->isAllowed( 'editinterface' ) ) ) ||
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Reduce the number of conditional operators (4) used in the expression (maximum allowed 3). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1067)